layouts/index.html

```diff
@@ -136,6 +136,12 @@ <h1>{{ .Params.welcomeheadline }}</h1>
     </a>
 </div>
 
+<div class="m-home-splash">
+    <a href="#">
```

Comment:
Thanks! Could you add border-radius of 6px on the link. You would also have to set it to display block and overflow hidden for the radius to work I think.

This would align its style with the other blocks.

Reply:
As this is a fairly temporary thing you can go ahead and add this style inline.